arwen/common.go

```diff
@@ -25,6 +25,8 @@ const (
 	BreakpointOutOfGas
 )
 
+const TimeLockKeyPrefix = "timelock"
```

Comment:
We now have the "ELROND" key prefix, which is reserved and prevents writing via storageStore. It might make sense to use it here too. Should be merged in master today, I'll follow up with a comment on how to use

Reply:
storageContext.elrondProtectedKeyPrefix contains this "ELROND" prefix and is configured from the protocol.
You could append something to it to get "ELRONDtimelock...." or "ELRONDtl...."

Reply:
I don't think you can write a key with the ELROND prefix from a SmartContract - the `StorageUpdate` that does this is ignored by the `scProcessor`, if I remember correctly.

Anyway, `timelock***` keys are fine, they are just a convenience tool, and this prefix is mere convention. The SC developer may choose to re-implement the timelock mechanism on their own. Storage of the timelocks should not protected by the protocol in a special way, they're just key/value pairs in the storage of the SC.